package.json

```diff
@@ -11,44 +11,44 @@
     "dependencies": {
         "aperture-config": "git+https://github.com/joyent/aperture-config.git#448811fa1696cd6c63d8d3391187c084b39b2e7b",
         "artedi": "1.1.1",
-        "assert-plus": "0.1.5",
-        "backoff": "2.3.0",
-        "bunyan": "0.22.1",
+        "assert-plus": "1.0.0",
+        "backoff": "2.5.0",
+        "bunyan": "1.8.12",
         "cmdln": "4.3.0",
         "content-disposition": "0.5.3",
-        "cueball": "2.2.9",
-        "dashdash": "1.3.2",
+        "cueball": "2.10.0",
+        "dashdash": "1.14.1",
         "deep-equal": "0.0.0",
-        "dtrace-provider": "0.2.8",
-        "http-signature": "1.1.0",
+        "dtrace-provider": "0.8.8",
+        "http-signature": "1.3.2",
         "jsprim": "^1.3.1",
-        "kang": "1.1.0",
+        "kang": "1.3.0",
         "keep-alive-agent": "0.0.1",
         "keyapi": "git+https://github.com/joyent/keyapi.git#e14b3d58",
         "libmanta": "git+https://github.com/joyent/node-libmanta.git#v1.3.0",
```

Comment:
I'm surprised that v1.4.0 of libmanta isn't required to get this:

```
commit 644c41b3f88b961f78c26f414494c2473d552f70 (tag: v1.4.0)
Date:   2019-09-06T15:35:31-04:00 (7 months ago)

    MANTA-4520 node-libmanta transitive dtrace-provider depdencies too old to build on x86_64
```

Reply:
That is interesting... I can still bump this de as it looks like `v1.3.0` -> `v.1.4.0` was only that change specifically.